tests/test_customer_subscriptions.py

```diff
@@ -90,17 +95,18 @@ def test_customer_subscription_get_related_customer(client, response):
 
 
 def test_cancel_customer_subscription(client, response):
-    """Cancel a subscription by customer ID and subscription ID"""
+    """Cancel a subscription by customer ID and subscription ID."""
     response.delete('https://api.mollie.com/v2/customers/%s/subscriptions/%s' % (CUSTOMER_ID, SUBSCRIPTION_ID),
                     'subscription_canceled', 200)
 
     subscription = client.customer_subscriptions.with_parent_id(CUSTOMER_ID).delete(SUBSCRIPTION_ID)
+    assert isinstance(subscription, Subscription)
```

Comment:
It seems that assertions like these are applied in a lot of places, but there are also some test where they are not added, such as `tests.test_payment_refunds.test_get_refund`.